bastion-gui.py

```
@@ -231,19 +231,42 @@ def handle_notification(self, req):
+            permanent = dialog.permanent
+
+            # Send response with decision_id
+            if self.connected and self.sock:
+                resp = json.dumps({
+                    'allow': decision,
+                    'permanent': permanent,
+                    'decision_id': decision_id
+                }) + '\n'
+                try:
+                    self.sock.sendall(resp.encode())
+                except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)